docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -10,7 +10,7 @@ described here, please review open issues in the following GitHub repositories:
 * https://github.com/elastic/kibana/issues[{kib}]
 * https://github.com/elastic/beats/issues[{beats}]
 * https://github.com/elastic/package-registry/issues[{package-registry}]
-* https://github.com/elastic/endpoint/issues[Elastic {endpoint-sec}]
+* https://github.com/elastic/endpoint/issues[{elastic-sec}]
```

Comment:
What repo are we trying to link to here? This `endpoint` repo isn't across all of security, though I'll need to check on the status. We track issues in https://github.com/elastic/security-team/issues but it's a private repo.

Reply:
I was looking for a public repo where users can see issues, but the endpoint repo does not seem to be used.

'll remove the link since it's not being used frequently anyhow. Just let me know if you think I should add it back.